src/lib/Addresses.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+    import { NETWORK } from "./config";
+    import EtherscanLink from "./EtherscanLink.svelte";
+
+    /**
+     * Displays the `owner` address of the RenPool.
+     */
+    export let owner: string;
+
+    /**
+     * Displays the `nodeOperator` address of the RenPool
+     */
+    export let nodeOperator: string;
+</script>
+
+<section>
+    <h2>Addresses</h2>
+    <div class="underline-title" />
+
+    <table class="w-full text-left border-collapse">
+        <thead>
+            <tr>
+                <th
+                    ><div class="pb-2 pr-2 border-b border-gray-200">
+                        Contract or Account
+                    </div></th
+                >
+                <th><div class="pb-2 border-b border-gray-200">Address</div></th
+                >
+            </tr>
+        </thead>
+        <tbody class="align-baseline">
+            <tr>
+                <td class="key">RenToken</td>
+                <td class="value"
+                    ><EtherscanLink address={NETWORK.contracts.REN_TOKEN} /></td
+                >
+            </tr>
+            <tr>
+                <td class="key">RenPool</td>
+                <td class="value"
+                    ><EtherscanLink address={NETWORK.contracts.REN_POOL} /></td
+                >
+            </tr>
+            <tr>
+                <td class="key">Owner</td>
+                <td class="value"><EtherscanLink address={owner} /></td>
+            </tr>
+            <tr>
+                <td class="key">Node Operator</td>
+                <td class="value"><EtherscanLink address={nodeOperator} /></td>
+            </tr>
+        </tbody>
+    </table>
+
+</section>
+
+<style lang="postcss">
+    h2 {
```

Comment:
Instead of defining the style of the font on each component we use it, I would rather introduce a `<Text/>` component with pre-defined styles for all common tags (p, h1, h2). I'd go something along these lines: 

https://github.com/fede-rodes/ballboy_client/blob/master/src/Themes/Fonts.ts
https://github.com/fede-rodes/ballboy_client/blob/master/src/Components/Common/Text/index.tsx

Related to this, we could pass an `underline` prop to the Text component so that every time we do `<Text tag="h2" {underline} />` we get the style we want without having to define the custom `underline-title` class

Reply:
I can't see the provided links, I see 404.

There is definitely space for refactor and follow a DRY approach. But keep in mind that everything there is configurable through tailwind. I would leave the styling to tailwind.

Reply:
Sorry, the repo was private. You should now be able to access the links.

Regarding your comments, agree on relying on tailwind but I think we could move the tailwind configuration into the `<Text />` component. In my view, the font style is part of the app's global theme, ie, we should define all styles we want to use across the app in one place and then make use of some reusable component